integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -294,6 +294,7 @@ def test_mod_pmod_by_zero(data_gen, overflow_exp):
         ansi_enabled_conf,
         exception_str)
 
+@pytest.mark.skipif(is_spark_314() or is_spark_322(), reason="Spark 3.1.4 and 3.2.2 will return null instead of throwing exception")
```

Comment:
what about spark 33X, 34X? it would be nice to have better context for this condition

Reply:
Updated